atc/creds/keyvault/reader.go

```diff
@@ -0,0 +1,111 @@
+package keyvault
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/keyvault/v7.0/keyvault"
+	"github.com/Azure/go-autorest/autorest"
+)
+
+// ErrSecretNotFound is the error returned when a secret cannot be found by the
+// reader
+var ErrSecretNotFound = fmt.Errorf("secret not found")
```

Comment:
Looks like this is unused? Probably 'cause `Get` just returns a `nil, false, nil` in this case anyway.